conductor/lib/common.py

```diff
@@ -440,14 +444,20 @@ def validate_api_key(config):
         Returns: None
 
         """
-        if 'api_key_path' not in config:
-            config['api_key_path'] = os.path.join(base_dir(), 'auth', 'conductor_api_key')
-        api_key_path = config['api_key_path']
+        # If the key isn't defined, than check for a path with the key
+        if 'api_key' in config:
+            logger.debug("'api_key' is already defined. Ignoring 'api_key_path'")
+            return
+
+        if 'api_key_path' not in config:          
+            api_key_path = Config.get_default_api_key_path()
+            logger.debug("'api_key_path' not found in config, checking base dir ({}) for api key path".format(api_key_path))
```

Comment:
The `info` log level is better suited for this because the default log level is above `debug` and won't be displayed when it would be helpful to know what path was used instead.